src/actions/read/auctions.ts

```diff
@@ -94,17 +99,17 @@ export const getAuction = (
   // get all the prices for the auction
   const prices = getAuctionPricesForInterval({
     startHeight: new BlockHeight(startHeight),
-    startPrice, // TODO: use IO class class
-    floorPrice,
+    startPrice: new mIOToken(startPrice), // TODO: use IO class class
+    floorPrice: new mIOToken(floorPrice),
     blocksPerInterval: 30, // TODO: this could be an input on the function
     auctionSettings: AUCTION_SETTINGS,
   });
 
   // calculate the minimum bid
   const minimumBid = calculateAuctionPriceForBlock({
     startHeight: new BlockHeight(startHeight),
-    startPrice,
-    floorPrice,
+    startPrice: new mIOToken(startPrice), // TODO: use IO class class
```

Comment:
Is this classy TODO now covered?

Reply:
[60a3aa7](https://github.com/ar-io/testnet-contract/pull/214/commits/60a3aa700c50a8887bbd557315c1d5b263ecc166)